docs/app/Examples/modules/Dropdown/Usage/DropdownExampleCustomSearchFunction.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import _ from 'lodash'
+import { Dropdown } from 'semantic-ui-react'
```

Comment:
Let's sort imports in alphabetical order

Reply:
Of course! :smile: On it!

Is there a linting rule for this that I'm missing?

Reply:
Nope, we don't have or don't use it :thinking:

Reply:
Alright. Btw, how do you sort it?  Alphabetically by module name, i.e. ```react```, ```lodash```, and ```semantic-ui-react```?

Reply:
--> ```lodash```, ```react```, ```semantic-ui-react```

Reply:
Yes, by module name.

Reply:
Thanks for updating.  FWIW, there is a fixable lint rule for this that we should turn on [`sort-imports`](https://eslint.org/docs/rules/sort-imports) so this is done automatically for people.

Reply:
@levithomason It seems that lint does not sort on the module name, but rather on the imported member, or am I missing something?